{{ cookiecutter.project_slug }}/docker/Dcokerfile.dev

```diff
@@ -0,0 +1,8 @@
+FROM {{ cookiecutter.project_slug }}.base
```

Comment:
If following Makefile, it should be `FROM {{cookiecutter.project_slug}}-image-base`.

Reply:
👍